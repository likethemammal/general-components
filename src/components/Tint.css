.container {
    position: absolute;
    top: 0;
    bottom: 0;
    left: 0;
    right: 0;
    z-index: 0;
    background: linear-gradient(0deg, rgba(0,0,0,0.6), transparent);
}