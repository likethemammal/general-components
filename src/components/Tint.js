import React, { Component } from 'react'

import css from './Tint.css'

export default class Tint extends Component {

    static defaultProps = {
        className: css.container,
    }

    render() {

        const { className } = this.props

        return <div className={className}></div>
    }
}