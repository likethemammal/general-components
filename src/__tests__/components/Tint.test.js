import React from 'react'
import {shallow} from 'enzyme'

import shared from '../_shared/tests'
import Tint from '../../components/Tint'


const component_jsx = <Tint />

describe('Tint', () => {
    // Render a checkbox with label in the document
    const component = shallow(
        component_jsx,
    )

    shared.SNAPSHOTS_SHOULD_MATCH([
        component,
    ])
})