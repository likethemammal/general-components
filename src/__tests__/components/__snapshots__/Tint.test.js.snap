// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tint Snapshot 0 should match 1`] = `<div />`;
